tools/install_demo_configuration.sh

```diff
@@ -1,6 +1,6 @@
-#!/bin/bash
-#install_demo_configuration.sh [-y]
+#!/usr/bin/env bash
 
+default_certs_expiry_days=365
```

Comment:
If we are setting this value to a `long time` could we make it 100 years worth of days instead.

Reply:
Actually I wanted to keep it short to encourage certificate rotation. 1 year seemed like a good balance.